src/Compatibility/Microsoft.DotNet.ApiCompat.Task/ValidateAssembliesTask.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.NET.Build.Tasks;
+using Microsoft.DotNet.ApiCompatibility.Logging;
+
+namespace Microsoft.DotNet.ApiCompat.Task
+{
+    public class ValidateAssembliesTask : TaskBase
+    {
+        /// <summary>
+        /// The assemblies that represent the contract.
+        /// </summary>
+        [Required]
+        public string[]? LeftAssemblies { get; set; }
+
+        /// <summary>
+        /// The assemblies that act as the implementation.
+        /// </summary>
+        [Required]
+        public string[]? RightAssemblies { get; set; }
+
+        /// <summary>
+        /// The path to the roslyn assemblies that should be loaded.
```

Comment:
This kind of hints me to believe that this should be a required argument, yet it is in reality optional. Either remove the Required attribute, specify more info on the comment saying we allow null, or both.

Reply:
Agree. The existing targets always provide the path to the roslyn assemblies but as you mentioned, in reality it isn't required. Applied your feedback.